src/mayara_pi.cpp

```
@@ -1464,6 +1464,21 @@ void mayara_pi::SyncLocalServerUrl() {
   m_client->SetLocalUrl(use_local ? MayaraServer::LocalUrl() : std::string());
 }
 
+// "Run it here" can be switched off from more than one place -- the Settings
+// dialog, and the search dialog's embedded MayaraServerPanel -- so both call
+// this rather than one of them repeating the cleanup. Without it, the
+// address remembered for a fast reconnect stays the loopback one of a server
+// we just stopped: every future session tries -- and fails against -- a
+// server that was deliberately turned off, before it falls through to real
+// discovery.
+void mayara_pi::ForgetLocalServerIfDisabled() {
+  if (!m_server || m_server->Enabled()) return;
```

Clang (14.0.6)

[warning] statement should be inside braces

(readability-braces-around-statements)